site/pages/api/duplicate-disruption.api.ts

```diff
@@ -36,10 +37,12 @@ const duplicateDisruption = async (req: NextApiRequest, res: NextApiResponse): P
 
         const newDisruptionId = randomUUID();
 
+        const displayId = cryptoRandomString({ length: 6 });
         const draftDisruption: Disruption = {
             ...validatedDisruptionBody.data,
             publishStatus: PublishStatus.draft,
             disruptionId: newDisruptionId,
+            displayId,
```

Comment:
Dont think this line is needed

Reply:
wanted to ensure all the data here is updated